### PR TITLE
README.md: add links to issue types and issue templats

### PR DESCRIPTION
**Pull request by macpijan**

### CODE REVIEW

**Review by dpsmith — APPROVED** [review]

**Review by pietrushnic — CHANGES_REQUESTED** [review]

**Comment by pietrushnic** [comment]
@macpijan it would be great to merge it before FOSDEM24

File: README.md [hunk 1]
```diff
@@ -1,21 +1,25 @@
...
-This repository tracks issues and feature development progress for the TrenchBoot
- Project. Please take a moment to read the following important information:
+This repository tracks issues and feature development progress for the
+[TrenchBoot Project](https://trenchboot.org/). Please take a moment to read the
+following important information:
 
 * [TrenchBoot Home](https://trenchboot.org)
 * [TrenchBoot Project](https://github.com/TrenchBoot)
 * [Documentation](https://github.com/TrenchBoot/documentation)
 * [Contributing](https://github.com/TrenchBoot/documentation/blob/master/CONTRIBUTING.md)
```
[the remark this answers is not in the document]
Comment (krystian-hebel) [reply README.md:10]: @macpijan ping

[the remark this answers is not in the document]
Comment (SergiiDmytruk) [reply README.md:11]: I've added it: https://github.com/TrenchBoot/trenchboot-issues/pull/7/commits/8015ee786cd536cf4a30dea87fabee6d462026f5

[the remark this answers is not in the document]
Comment (SergiiDmytruk) [reply README.md:11]: @pietrushnic, ping.

**Review by pietrushnic — APPROVED** [review]

**Pull request merged**